Pipfile

```diff
@@ -10,6 +10,7 @@ pytest = "*"
 pytest-cov = "*"
 
 [packages]
+numpy = "~=1.20.2"
```

Comment:
can you please explain why `numpy` is  being added in this PR even though there is no added python code that uses `numpy`